### PR TITLE
[Docs] Update Package.swift docs

### PR DESCRIPTION
**Pull request by ncooke3**
#no-changelog

### CODE REVIEW

**Review by paulb777 — APPROVED** [review]

File: Package.swift [hunk 1]
```diff
@@ -29,12 +29,14 @@ let package = Package(
       name: "FirebaseAnalytics",
       targets: ["FirebaseAnalyticsTarget"]
     ),
-    // This library is not designed to be imported into client source code.
+    // Adding this product to the your project is enough for it to take effect– the module
+    // does not need to be imported into any source files.
```
Comment (ncooke3) [inline Package.swift:33]:
```suggestion
    // Adding this library to the your project is enough for it to take effect. The module
    // does not need to be imported into any source files.
```

File: Package.swift [hunk 2]
```diff
@@ -29,12 +29,14 @@ let package = Package(
...
     .library(
       name: "FirebaseAnalyticsWithoutAdIdSupport",
       targets: ["FirebaseAnalyticsWithoutAdIdSupportTarget"]
     ),
-    // This library is not designed to be imported into client source code.
+    // Adding this product to the your project is enough for it to take effect– the module
+    // does not need to be imported into any source files.
```
Comment (ncooke3) [inline Package.swift:39]:
```suggestion
    // Adding this library to the your project is enough for it to take effect. The module
    // does not need to be imported into any source files.
```

File: Package.swift [hunk 3]
```diff
@@ -29,12 +29,14 @@ let package = Package(
       name: "FirebaseAnalytics",
       targets: ["FirebaseAnalyticsTarget"]
     ),
-    // This library is not designed to be imported into client source code.
+    // Adding this product to the your project is enough for it to take effect– the module
```
[the remark this answers is not in the document]
Comment (ncooke3) [reply Package.swift]: Done!

**Review by andrewheard — APPROVED** [review]

File: Package.swift [hunk 4]
```diff
@@ -29,12 +29,14 @@ let package = Package(
       name: "FirebaseAnalytics",
       targets: ["FirebaseAnalyticsTarget"]
     ),
-    // This library is not designed to be imported into client source code.
+    // Adding this library to the your project is enough for it to take effect. The module
```
Comment (rizafran) [inline Package.swift:32]: "Adding this library **to your** project is enough for it to take effect. The module"
or
"Adding this library **to the** project is enough for it to take effect. The module"

File: Package.swift [hunk 5]
```diff
@@ -29,12 +29,14 @@ let package = Package(
...
+    // does not need to be imported into any source files.
     .library(
       name: "FirebaseAnalyticsWithoutAdIdSupport",
       targets: ["FirebaseAnalyticsWithoutAdIdSupportTarget"]
     ),
-    // This library is not designed to be imported into client source code.
+    // Adding this library to the your project is enough for it to take effect. The module
```
Comment (rizafran) [inline Package.swift:38]: "Adding this library **to your** project is enough for it to take effect. The module"
or
"Adding this library **to the** project is enough for it to take effect. The module"

File: Package.swift (hunk 4 above)
Comment (ncooke3) [inline Package.swift:32]:
```suggestion
    // Adding this library to your project is enough for it to take effect. The module
```

File: Package.swift (hunk 5 above)
Comment (ncooke3) [inline Package.swift:38]:
```suggestion
    // Adding this library to your project is enough for it to take effect. The module
```

Comment (ncooke3) [reply Package.swift]: good catch, thanks!

**Review by rizafran — APPROVED** [review]

**Pull request merged**